trafficman.tf

```diff
@@ -5,7 +5,7 @@ data "template_file" "tmtemplate" {
 resource "azurerm_template_deployment" "tmprofile" {
   template_body       = "${data.template_file.tmtemplate.rendered}"
   name                = "${var.product}-${var.env}-tm"
-  resource_group_name = "${local.resource_group_name}"
+  resource_group_name = "${azurerm_resource_group.rg.name}"
```

Comment:
please revert this change, we're in the progress of migrating applications to always passing in a resource group to the module, as we're decommissioning this module as part of moving to Kubernetes

REQ

Reply:
https://github.com/hmcts/cnp-module-webapp/pull/156#discussion_r322741012